### DISCUSSION TITLE
Is it possible to show current version in URL path?

### DISCUSSION THREAD

**Discussion by imballinst** [facebook/docusaurus#5429] [Q&A] [answered]
Hello! I'm wondering if it's possible to set the current version to have an explicit version in the URL path as well? For example, at the moment in the Docs site:

**2.0.0-beta4:** https://docusaurus.io/docs
**2.0.0-beta3**: https://docusaurus.io/docs/2.0.0-beta.3

Would it be possible if the version **2.0.0-beta4** to have URL path https://docusaurus.io/docs/2.0.0-beta.4? Thanks!

**Comment by slorber** [comment, marked as answer]
Yes the docs plugin has a `versions` option allowing to customize many things on a per-version basis, including paths.

Also, make sure to choose the appropriate versioning behavior:

https://docusaurus.io/docs/versioning#figure-out-the-behavior-for-the-current-version

**Reply by imballinst** [reply]
Thanks! I read that section but I haven't tried it yet (since I was unsure if it was the solution I sought). But, now, I'll definitely give it a try. Thanks again for replying and for the reference!
